README.md

```diff
@@ -0,0 +1,85 @@
+# ReactJS Boilerplate
+
+## Roadmap
+
+Our roadmap for the ReactJS Boilerplate is where you can learn about what features we're working on.
+
+### Development environment
```

Comment:
@jinmayamashita I think we can split folders into modules like the admin template

Reply:
@mvn-luanvu-hn It seems like a good thing to consider. Let's keep discussing the folder structure!